src/app.html

```diff
@@ -17,7 +17,7 @@
     <a class="nav-link" id="nav-about" href="#/about">About</a>
     <a class="nav-link open" id="nav-header" href="#/header">Header</a>
     <a class="nav-link" id="nav-mime" href="#/mime">MIME types</a>
-    <a class="nav-link" id="nav-security" href="#/security">Security</a>
+    <a class="nav-link" id="nav-security-definitions" href="#/securityDefinitions">Security</a>
```

Comment:
I am not sure that we need to change the url slug. 'security' seems fine. Also, we don't usually use camel case in URLs.